testsuite/testcases/src/compatibility_test.rs

```diff
@@ -51,54 +52,72 @@ impl NetworkTest for SimpleValidatorUpgrade {
         let duration = Duration::from_secs(5);
 
         let msg = format!(
-            "1. Downgrade all validators to older version: {}",
+            "1. Check liveness of validators at old version: {}",
```

Comment:
Who is checking liveness of the validators here?

Reply:
**liveness of the entire chain ig. that the chain is making progress and validators are committing new versions